### DISCUSSION TITLE
windows architecture is not supported by delve

### DISCUSSION THREAD

**Discussion by Vlad-ang** [go-delve/delve#3354] [Q&A] [answered]
Help me plz,
OS: Windows 10
PS D:\Me>    go clean -modcache
PS D:\Pars\Bots> go install github.com/go-delve/delve/cmd/dlv@latest
go: downloading github.com/go-delve/delve v1.20.2
go: downloading github.com/sirupsen/logrus v1.6.0
go: downloading github.com/spf13/cobra v1.1.3
go: downloading github.com/mattn/go-isatty v0.0.3
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/cosiner/argv v0.1.0
go: downloading github.com/derekparker/trie v0.0.0-20221213183930-4c74548207f4
go: downloading github.com/go-delve/liner v1.2.3-0.20220127212407-d32d89dd2a5d
go: downloading github.com/mattn/go-colorable v0.0.9
go: downloading github.com/google/go-dap v0.7.0
go: downloading github.com/hashicorp/golang-lru v0.5.4
go: downloading golang.org/x/arch v0.0.0-20190927153633-4e8777c89be4
go: downloading go.starlark.net v0.0.0-20220816155156-cfacd8902214
go: downloading golang.org/x/sys v0.0.0-20220908164124-27713097b956
go: downloading github.com/cilium/ebpf v0.7.0
go: downloading github.com/mattn/go-runewidth v0.0.13
go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.3
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.0
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: downloading github.com/rivo/uniseg v0.2.0
go: downloading github.com/russross/blackfriday/v2 v2.0.1
go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
C:\Users\Me\go\pkg\mod\github.com\go-delve\delve@v1.20.2\service\debugger\debugger.go:32:2: found packages native (followexec_other.go) and your_windows_architecture_is_not_supported_by_delve (support_sentinel_windows.go) in C:\Users\Me\go\pkg\mod\github.com\go-delve\delve@v1.20.2\pkg\proc\native

**Comment by alexsaezm** [comment]
Which architecture?

**Reply by Vlad-ang** [reply]
Go x86

**Reply by alexsaezm** [reply]
Do you mean 32bits?

**Reply by Vlad-ang** [reply]
I tryed with x86-64 and just x86, with both of them i had errors

**Reply by Vlad-ang** [reply]
64-bit system

**Reply by alexsaezm** [reply, marked as answer]
I don't have a Windows machine at hand but Go and Delve both support x86_64. From the prompt of your initial message, I see a bunch of different paths, you should check that your Go installation is properly configured.

**Reply by Vlad-ang** [reply]
I reinstalled Go to x86_64 and it solved my problem.
